integration-test-standard/src/test/java/com/ericsson/bss/cassandra/ecaudit/integration/standard/ITDataAudit.java

```diff
@@ -103,7 +103,7 @@ public static void afterClass()
             new Object[]{ "ALTER KEYSPACE dataks WITH DURABLE_WRITES = false", "alter", "data/dataks" },
             new Object[]{ "DROP TYPE IF EXISTS dataks.tp", "drop", "data/dataks" },
             new Object[]{ "DROP MATERIALIZED VIEW dataks.viw", "alter", "data/dataks/tbl" },
-            new Object[]{ "DROP INDEX IF EXISTS dataks.idx", "alter", "data/dataks/tbl" },
+            new Object[]{ "DROP INDEX IF EXISTS dataks.idx", "alter", "data/dataks/idx" },
```

Comment:
This (and the create index statement above) seem a bit wiered.

At the very least this is an unfortunate difference compared to 3.11. But honestly, I believe this is wrong. For instance, would you be able to tell the difference on this vs. altering a table named "idx" in keyspace dataks?

So, either we should try to preserve existing functionality, or perhaps explore if this should be mapped to another resource type? Perhaps "index" instead of "data". Is there perhaps a type to exploint in 4.0 codebase?

Reply:
Yes, this is a bit wired, it seams to be a change in Cassandra 4.0 but I have not found the commit that changes this, so it needs a bit more investigation.
But I think that behind the scenes in Cassandra that "DROP INDEX dataks.idx" is a "DROP TABLE dataks.idx" so it might not be any difference.
I'm not sure if we can easily preserve the functionality or if we want to, if this is a change in 4.0 we might not want to hide that.

Reply:
I have looked into this a bit more and I think I understand whats happening, not sure how to solve it yet but I think I agree with you, it should not be changed to "idx" it should still be "tbl".
We solve a similar issue with views but the solution here seems to be different.

Reply:
OK, got it.

Then again, to put things in perspective, perhaps this isn't a deal breaker. If this is blocking progress, then it might be OK to just create a ticket for this and perhaps list it as a known issue in documentation.

Reply:
I have solved the issue now.